src/types/tests/schema-test.ts

```diff
@@ -23,6 +23,7 @@ import {Entity} from '../../runtime/entity.js';
 import {ConCap} from '../../testing/test-util.js';
 import {Flags} from '../../runtime/flags.js';
 import {deleteFieldRecursively} from '../../utils/lib-utils.js';
+import {SchemaFieldKind} from '../../runtime/manifest-ast-types/manifest-ast-nodes.js';
```

Comment:
Is this import doing anything? Looks like it's not needed

Reply:
Thanks, it got copied when I was unstacking my branches, I'll remove it.

Reply:
The only usage of `SchemaFieldKind` that I can see in this file is this import statement: https://github.com/PolymerLabs/arcs/blob/3761cac3b045bfe13bf501dc8e193b0499a998cd/src/types/tests/schema-test.ts

Is there a missing commit or something?

Reply:
Sorry @shans and @csilvestrini it seems I hadn't properly removed code that I will add later (so, it will be used, but isn't yet).

Thanks, removed it now.